Services/Interfaces/web_interface/api/config.py

```diff
@@ -45,3 +45,16 @@ def change_reference_market_on_config_currencies():
     success, reply = models.change_reference_market_on_config_currencies(request_data["old_base_currency"],
                                                                          request_data["new_base_currency"])
     return util.get_rest_reply(flask.jsonify(reply)) if success else util.get_rest_reply(reply, 500)
+
+
+@api.api.route('/copy_trading_strategy', methods=["POST"])
+@login.login_required_when_activated
+def copy_trading_strategy():
+    try:
+        product_slug = flask.request.get_json()["product_slug"]
```

Comment:
I think we shouldn't call it product_slug because it could be something else in the future. I think we should consider this data as for example the "copy_id"

Reply:
yes !